notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        display_name="Salads" + "_" + "unique",
</pre><p>Just write: "salads_unique"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>renamed to salads_unique</p>